pvlib/modelchain.py

```diff
@@ -577,7 +582,10 @@ def infer_aoi_model(self):
             return self.ashrae_aoi_loss
         else:
             raise ValueError('could not infer AOI model from '
-                             'system.module_parameters')
+                             'system.module_parameters. Check that the '
+                             'parameters contain physical, aoi, or ashrae '
```

Comment:
`system.module_parameters contain parameters for the physical, aoi, or ashrae model; explicitly set model with aoi_model kwarg; or`